### ISSUE TITLE
METAR based on coordinates: Reporting vs Non-reporting Stations

### ISSUE THREAD

**Issue by vikaask** [avwx-rest/AVWX-API#18]
METAR lookup based on coordinates pulls up non-reporting stations instead of prioritizing reporting stations. 

Unable to use Reporting flag on search as well.

**Comment by vikaask** [comment]
In the past, the following lookup 

> **https://avwx.rest/api/metar/33.2,-111.7?onfail=&options=info&format=json**

would provide a METAR for KCHD airport. In the last 2 days, the same lookup is now trying to pull up a METAR from a non-reporting airport

> {
>   "error": "5AZ3 does not publish reports",
>   "timestamp": "2019-08-24T05:46:42.886751Z"
> }
> 


This basically renders the Coordinates functionality useless as it would require being at the exact coordinates of a reporting airport to receive METAR data.

**Comment by flyinactor91** [comment]
I have "reporting" and "airport" filters available for nearest station requests, so the logic is there. I'll need to change how parameters are validated to make this apply to report-based coordinate lookups.

**Comment by vikaask** [comment]
Yeah, strange though how coordinate lookups as recent as last week worked as expected. Any recent changes? I assume it's something related to the airport database changing to include those stations? 

Sorry I'm pretty new to this so I'm still wrapping my brain around this stuff

**Comment by flyinactor91** [comment]
Yes the station database has been expanded to include all non-closed
airports, seaplane bases, helipads, and remote weather stations. I used to
pass the ident on the report services as is, but the expanding info
database and desire to cut out non-existent station calls led me to
validate idents against a comprehensive list.

The problem right now is that the idents and coordinates are converted at
parameter validation. I’d need to move station validation later or the
filters earlier in order to make that happen. I might have time tomorrow.
Otherwise it will have to wait until later in the week.

**Comment by tthamel** [comment]
wait what dude is based out of chandler ? 😂
